arroyo-sql/src/lib.rs

```diff
@@ -180,13 +203,26 @@ impl ArroyoSchemaProvider {
             };
 
             let mut args: Vec<TypeDef> = vec![];
+            let mut vec_arguments = 0;
             for (i, arg) in function.sig.inputs.iter().enumerate() {
+                info!("arg {}: {:?}", i, quote::quote!(#arg).to_string());
```

Comment:
this is a bit noisy — maybe downgrade to debug?

Reply:
These were just me trying to understand the existing code, will delete.